docs/postgresql.md

```diff
@@ -7,15 +7,30 @@ for instructions to collect logs and metrics from this application using Ops Age
 
 The Ops Agent collects the following metrics from your postgresql instances.
```

Comment:
The `docs` folder can probably be deleted now that we track everything in `metadata.yaml`. [Documentation](https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/third-party) is automatically generated based on `metadata.yaml` so we don't need to manually update the `.md` files in this repo any more.

CC: @sophieyfang for confirmation

Reply:
Confirmed with @sophieyfang that they can be deleted. Sending a separate PR to delete the whole folder. For this PR, we can just ignore this file.

Reply:
Sent https://github.com/GoogleCloudPlatform/ops-agent/pull/842